wf.md

```diff
@@ -0,0 +1,82 @@
+[Powrót](README.md)
+
+# WF
+**Dla studentów 1. roku:** na WF możecie zapisywać się dopiero od drugiego semestru, ale już na pierwszym możecie zapisać się do [sekcji sportowej](#sekcje-sportowe).
+
+## Wymagania
+Każdy musi zrealizować 60 godzin WF-u, co przekłada się na **2 semestry** zajęć (dowolne dwa w trakcie studiów).
```

Comment:
Może dodać tutaj (albo w Q&A, lub na górze w tej głównej sekcji?) informację o liczbie ECTS za wf (to chyba nadal jest 0?), żeby nie było nieporozumień?

Reply:
done